### PR TITLE
added code for polyndrome

### PR DESCRIPTION
**Pull request by shams-rasulzade**

### CODE REVIEW

File: src/Polyndrome_Numbers/Polyndrome.java [hunk 1]
```diff
@@ -1 +1,35 @@
...
+
+    public static boolean polyndrome(int n){
+
+        if (n < 0) return false;
+        int reverse = 0, original = n;
+
+
+        while (n != 0){
+            reverse = reverse * 10 + n%10;
+            n= n / 10;
```
Comment (sabuhihuseynov) [inline src/Polyndrome_Numbers/Polyndrome.java:15]: pls use ctrl alt l

File: src/Polyndrome_Numbers/Polyndrome.java [hunk 2]
```diff
@@ -1 +1,35 @@
...
\ No newline at end of file
+package Polyndrome_Numbers;
+
+import java.util.Scanner;
+
+public class Polyndrome {
+
+    public static boolean polyndrome(int n){
+
+        if (n < 0) return false;
```
Comment (sabuhihuseynov) [inline src/Polyndrome_Numbers/Polyndrome.java:9]: also consider n%10==0 which we discussed in class

**Review by sabuhihuseynov — CHANGES_REQUESTED** [review]
pls look

File: src/Polyndrome_Numbers/Polyndrome.java (hunk 2 above)
Comment (shams-rasulzade) [reply src/Polyndrome_Numbers/Polyndrome.java:9]: it's considered on while (n!==0)

**Pull request closed without merging**